### DISCUSSION TITLE
Only support loading uncompressed library?

### DISCUSSION THREAD

**Discussion by moeshin** [LSPosed/LSPosed#940] [Q&A] [answered]
https://github.com/LSPosed/LSPosed/blob/4e554affc325cb49ac3af422644d1fd130fe60ee/core/src/main/java/org/lsposed/lspd/util/LspModuleClassLoader.java#L100

https://github.com/LSPosed/LSPosed/blob/4e554affc325cb49ac3af422644d1fd130fe60ee/core/src/main/java/de/robv/android/xposed/XposedInit.java#L337-L342

**Comment by yujincheng08** [comment, marked as answer]
yes

**Reply by moeshin** [reply]
Ok. I add `android:extractNativeLibs="false"` in AndroidManifest.xml to prevent compression.
